components/calendar/src/iso.rs

```diff
@@ -457,6 +457,7 @@ impl Iso {
 
     // Lisp code reference: https://github.com/EdReingold/calendar-code2/blob/1ee51ecfaae6f856b0d7de3e36e9042100b4f424/calendar.l#L1237-L1258
     pub(crate) fn iso_from_fixed(date: i32) -> Date<Iso> {
+        // Temporary fix to visibility issue: don't leave, please address
```

Comment:
suggestion: remove comment if you already addressed issue

Is this comment actually related to the visibility of the function `iso_from_fixed`? (in other words, the `pub(crate)` vs. `pub` visibility modifier)

If so:

1. Remove, since the issue seems to be addressed
2. Btw, comments that take up their own line(s) usually apply to the lines that follow, not the lines before.
3. You want the comment to be more precisely worded so that they make sense even outside of the context of a review. So for example:

```
// TODO(#<issuenumber>): Temporarily changed pub to pub(crate) as a workaround for visibility issue
```

Reply:
Thanks, this issue was already fixed and I have deleted the comment in the latest commit